### DISCUSSION TITLE
Expérimentation végé 🥬

### DISCUSSION THREAD

**Discussion by alemangui** [betagouv/ma-cantine#1228] [Autres sujets]
Après discussion avec @valeriemerle et @ErwanGv voici un post pour peaufiner les questions et textes de l'expérimentation végétarienne.

## Texte d'introduction (avant d'ouvrir le formulaire)

----------------------
> #### Expérimentation menu végetarien
> Pour évaluer les effets des menus végéteriens sur la satisfaction de vos convives et le taux de fréquentation de votre établissement, nous vous proposons de participer à une expérimentation prévue par la loi climat et résilience.
> - [ ] Je suis volontaire pour participer à l'expérimentation 
-----------------------

## Questions

1- Case à cocher avec sélecteur de date en cas de case cochée

----------------------
> - [x] J'ai déjà mis en place l'option végétarienne quotidienne
> Date de mise en place : _____________
----------------------

2- Taux de prise (T0 et T1)

----------------------
> Pourcentage de menus végétariens servis par rapport aux autres menus : ______ %
----------------------

3- Catégorisation sur 20 repas végétariens servis (T0 et T1)

----------------------
> Veuillez indiquer la composition principale sur 20 repas végétariens servis :

> 
> |  | Récurrence |
> |--------------|-----------|
> | Œufs (omelette, œuf dur...) | ____ sur 20 repas |
> | Fromage (pané fromager, tartiflette sans lardons...) | ____ sur 20 repas |
> | Galettes/boulettes/nuggets prêt à l’emploi à base de soja | ____ sur 20 repas |
> | Galettes/boulettes/nuggets prêt à l’emploi sans soja | ____ sur 20 repas |
> | Plats à base de céréales| ____ sur 20 repas |
> | Légumineuses et légumes (dahl, chili végétarien...) | ____ sur 20 repas |

----------------------

4- Gaspillage végé (T0 et T1)

----------------------
> Gaspillage des menus végétariens

> 
> |  | Quantité (gram / convive) |
> |--------------|-----------|
> | Reste en production (non servi) | ____ (g/convive) |
> | Reste de composantes (entrée, plat, dessert...) | ____ (g/convive) |

----------------------

5- Gaspillage non-végé (T0 et T1)

----------------------
> Gaspillage des menus non-végétariens

> 
> |  | Quantité (gram / convive) |
> |--------------|-----------|
> | Reste en production (non servi) | ____ (g/convive) |
> | Reste de composantes (entrée, plat, dessert...) | ____ (g/convive) |

----------------------

6- Évolution ressentie du gaspillage avec un pourcentage en cas de oui / non (T0 et T1)

----------------------
> Constatez-vous plus de gaspillage avec les options végétariennes ?

> 
> - [x] Oui, il y a plus de gaspillage
> - [ ] Non, il y a moins de gaspillage
> - [ ] Pas de différence notable
>
> Écart en pourcentage : ____ %
----------------------

6- Formation (T0 et T1)

----------------------
> Votre cantine a mis en place une formation spécifique des cuisiniers ou gestionnaires sur les menus végétariens ?
> 
> - [ ] Oui
> - [ ] Non
>
----------------------

7- Coût végé (T0 et T1)

----------------------
> Coût moyen du repas végétarien (en € / assiette) : ______ €
----------------------

8- Coût non-végé (T0 et T1)

----------------------
> Coût moyen du repas non-végétarien (en € / assiette) : ______ €
----------------------

9- Évolution du coût + pourcentage (T0 et T1)

----------------------
> Comment le coût facture aux familles a-t-il évolué depuis la mise en place du menu quotidien ?
>
> - [x] Le coût a augmenté
> - [ ] Le coût a diminué
> - [ ] Pas de différence notable
>
> Écart en pourcentage : ____ %
----------------------

10- Satisfaction convive (T0 et T1)

----------------------
> Satisfaction moyenne convive
> ☆☆☆☆☆
----------------------

## Temporalité

Formulaire sera rempli deux fois : un à T0 (libellé **À date**) et un autre T1 six mois après sans dépasser décembre 2022 (libellé **Six mois après**).

**Comment by alemangui** [comment]
@ErwanGv j'ai pris les catégories du mail retransféré par @valeriemerle.

N'hésitez pas à faire de retours concernant le type de données demandés (unités, questions, etc) ainsi que le wording.

**Reply by ErwanGv** [reply]
Merci Alejandro !
Ci-dessous ma liste de propositions (modifications **en gras**). N'hésite pas à revenir vers moi si ce n'est pas clair.

- Texte d'introduction (avant d'ouvrir le formulaire)
    
- [x]  Expérimentation **de l'option végétarienne quotidienne pour les collectivités volontaires** 
- [x] **Pour participer à l'expérimentation d'une option végétarienne quotidienne, telle que prévue par la loi Climat et résilience, nous vous proposons de répondre à quelques questions sur la mise en œuvre de l'option végétarienne quotidienne dans votre établissement. En particulier, les questions portent sur les catégories de plats végétariens servis, l'impact sur les pesées de gaspillage alimentaire, la fréquentation, la satisfaction des convives et le coût des repas. 
Il est également possible de participer si une option végétarienne quotidienne est déjà mise en place dans votre établissement.**
- [x] Ajouter les liens vers les livrables existants (en attendant le guide dédié) : **Consultez le livret de recettes végétariennes du CNRC**, lien : https://ma-cantine-1.gitbook.io/ma-cantine-egalim/diversification-des-sources-de-proteines-et-menus-vegetariens/guide-pour-la-mise-en-place-du-menu-vegetarien-en-milieu-scolaire et **Consultez le cadre pour la mise en oeuvre du plan pluriannuel de diversification des sources de protéines**, lien : https://ma-cantine-1.gitbook.io/ma-cantine-egalim/diversification-des-sources-de-proteines-et-menus-vegetariens/untitled

- Questions
3- Catégorisation sur 20 repas végétariens servis (T0 et T1)

- [x] Remplacer l'en-tête par : **Dans votre plan alimentaire de 20 repas successifs, les options végétariennes servies dans votre établissement sont des plats à base de :**
- [x] Pour les Galettes/boulettes/nuggets, il y a deux attributs indépendants : "fait maison/prêt à l'emploi" et "à base de soja/sans soja". Donc soit il faut mettre 4 lignes avec les 4 combinaisons, soit des menus déroulants pour choisir les deux attributs (tout en laissant la possibilité d'avoir plusieurs lignes).
- [x] Les deux dernières lignes sont une seule ligne : **plats à base de céréales, légumineuses et légumes (dahls, chili végétarien…)**

4- Gaspillage végé (T0 et T1) et 5- Gaspillage non-végé (T0 et T1)

- [x] Remplacer l'en-tête par : **Résultats des pesées de gaspillage alimentaire sur au moins 3 jours consécutifs (en g/convive)**
- [x] Rassembler les deux tableaux (une colonne **Repas végétariens** et une colonne **Repas non-végétariens**)
- [x] Pour harmoniser avec la solution de réservation, modifier les intitulés : **Moyenne des pesées des excédents présentés aux convives et non servis** et **Moyenne des pesées des restes des assiettes** 

6- Évolution ressentie du gaspillage avec un pourcentage en cas de oui / non (T0 et T1)

- [x] Remonter la question avant les questions 4 et 5

6- Formation (T0 et T1)

- [x] Descendre la question à la fin du questionnaire
- [x] Votre **établissement a-t-il** mis en place une formation spécifique des cuisiniers ou gestionnaires sur les menus végétariens ?

7- Coût végé (T0 et T1)

- [x] Ajouter en en-tête : **Le coût matière moyen des menus végétariens est-il supérieur aux autres menus ?**

9- Évolution du coût + pourcentage (T0 et T1)

- [x] Comment le coût factur**é** aux familles a-t-il évolué depuis la mise en place du menu quotidien ?

10- Satisfaction convive (T0 et T1)

- [x] **Quel est le niveau de** satisfaction **moyen des** convives ?
- [x] Ajouter une question : **Quelles sont les principales raisons évoquées ?** avec comme réponses (multiples possibles) : **Liberté de choix (régime, culte...)**,  **Goût et texture**, **Nouveauté**, **Variété des recettes**, **Méconnaissance**, **Opposition de principe**, **Impact sur la santé**, **Impact sur l'environnement**
- [x] Ajouter une question : **Quel est le niveau de satisfaction moyen du personnel ?**
- [x] Ajouter une question : **Quelles sont les principales raisons évoquées ?** avec comme réponses (multiples possibles) : **Liberté de choix (régime, culte...)**,  **Nouveauté**, **Variété des recettes**, **Méconnaissance des techniques de cuisine**, **Opposition de principe**, **Impact sur la santé**, **Impact sur l'environnement**

Nouvelle question : 

- [x] **Avez-vous utilisé le livret de recettes végétariennes publié par le CNRC ou par d’autres organismes ?**, réponse "Oui", "Non"

**Reply by alemangui** [reply]
Top merci @ErwanGv, je commence le dev et je reviens vers toi en cas de souci :+1:

**Reply by alemangui** [reply]
@ErwanGv pas sur de comprendre cette remarque : 

> 7- Coût végé (T0 et T1)
> - [ ] Ajouter en en-tête : Le coût matière moyen des menus végétariens est-il supérieur aux autres menus ?

"Ajouter en en-tête" çad comme un titre ? Ou bien on parle d'une question avec oui/non ?

Comme un titre ça donnerait :

---------------------------------------
> ##### Le coût matière moyen des menus végétariens est-il supérieur aux autres menus ?
> Coût moyen du repas végétarien (en € / assiette) : _________ euros
---------------------------------------

Comme une question additionnelle ça donnerait : 

---------------------------------------
> - [ ] Le coût matière moyen des menus végétariens est supérieur aux autres menus
>
> Coût moyen du repas végétarien (en € / assiette) : _________ euros
---------------------------------------

Tu préfères lequel des deux ?

**Reply by ErwanGv** [reply]
Bonjour Alejandro,
C'est la première option, comme un titre (avec les deux questions quantitatives 7 et 8 en dessous) : 
    Le coût matière moyen des menus végétariens est-il supérieur aux autres menus ?
    Coût moyen du repas végétarien (en € / assiette) : _________ euros
    Coût moyen du repas non-végétarien (en € / assiette) : ______ euros

Merci beaucoup !
Erwan

**Reply by alemangui** [reply]
Parfait, merci !

**Reply by alemangui** [reply]
Hello @ErwanGv :wave:  voici les questions sur le formulaire, en jetant un coup d'oeil rapide ça te semble correct ? je ne rate pas qqchose de gros non ?

(C'est juste la partie UI, il n'y a rien de connecté dans la base de données ni de validation encore, et il me faut aussi ajouter les liens vers les resources)

Merci !

![expe-vg](https://user-images.githubusercontent.com/1225929/160640314-831e57cb-fcd3-4688-91eb-29bea3c0e92f.gif)

**Reply by ErwanGv** [reply]
Merci beaucoup !
Je me rends compte qu'on a oublié les questions relatives à la fréquentation par rapport à la discussion hier. 

- [x] Il faudrait donc ajouter une question : 
**Constatez-vous plus une évolution de la fréquentation depuis l'introduction de l'option végétarienne quotidienne ?
-La fréquentation a augmenté
-La fréquentation a diminué
-Pas de différence notable
Écart en pourcentage : ____ %**

- [x] Et peut être simplement ajouter au début la date prévisionnelle de début de l'expérimentation, pour ceux qui ne l'ont pas déjà mise en place (comme pour la solution de réservation).

- [x] Enfin, comme discuté hier, pour le T1 nous pouvons indiquer "En décembre 2022".

Après ça on sera bons !

Erwan

**Reply by alemangui** [reply]
Ça marche :+1:  Les changements ont été faits, jeudi j'attaque la partie données. Merci !

![expe-vg-2](https://user-images.githubusercontent.com/1225929/160656080-05493280-4e16-43fd-a58d-0140f9449822.gif)

**Reply by ErwanGv** [reply]
Merci à toi, bonne soirée!

**Comment by alemangui** [comment]
Pull request en attente de revue de code : https://github.com/betagouv/ma-cantine/pull/1237

**Comment by alemangui** [comment]
Fonctionnalité disponible sur [staging](ma-cantine-staging.cleverapps.io/)
![expe-vg-staging](https://user-images.githubusercontent.com/1225929/162146205-fa06ab7e-4ee2-460f-a427-5061661f42ec.gif)

**Comment by ErwanGv** [comment]
Merci Alejandro !
Deux remarques dans un premier temps, avant les retours utilisateurs : 
- [x] Est-il possible de mettre le même texte introductif dans l'autodiagnostic et en haut de la fenêtre qui s'ouvre, à la place de "afin d'évaluer les effets..." : **Pour participer à l'expérimentation d'une option végétarienne quotidienne, telle que prévue par la loi Climat et résilience, nous vous proposons de répondre à quelques questions sur la mise en œuvre de l'option végétarienne quotidienne dans votre établissement. En particulier, les questions portent sur les catégories de plats végétariens servis, l'impact sur les pesées de gaspillage alimentaire, la fréquentation, la satisfaction des convives et le coût des repas.
Il est également possible de participer si une option végétarienne quotidienne est déjà mise en place dans votre établissement.**

- [ ] Est-ce que les champs de données peuvent être rendus obligatoires (un message qui empêche de valider le questionnaire si tous les champs ne sont pas remplis) ? J'ai un doute pour les pesées de gaspillage, mais je propose de les conserver en obligatoire dans un premier temps, et de voir les retours utilisateurs sur le sujet.

Merci d'avance !

Erwan

**Reply by alemangui** [reply]
Hello @ErwanGv, merci pour les remarques ! Le changement du texte introductif est en cours ([PR par ici](https://github.com/betagouv/ma-cantine/pull/1263)) ;)

Concernant les données obligatoires ça peut être un peu risqué par rapport à l'engagement. On a constaté quelques soucis lors qu'on rendait obligatoire les champs des formulaires pour lesquels les utilisateurs peuvent revenir dessus (comme les diagnostics). 

Souvent ils démarrent le remplissage et puis se rendent compte qu'une donnée leur manque. Ils continuent (en se disant qu'ils reviendront après la renseigner) et puis ils sauvegardent et mettent à jour plus tard. Lors qu'on rend les champs obligatoires ils ne peuvent pas sauvegarder (même partiellement) et ça provoque une frustration d'avoir rempli des champs pour rien, et souvent on perd cet engagement.

De plus, les champs numériques sont particulièrement délicats dessus, car certains utilisateurs décident de laisser le champ vide pour indiquer un "0". Par ex, si je n'ai pas de produits bio, je laisse le champ en blanc. Si on les met en obligatoire, ça provoque certaines confusions et l'engagement souffre aussi.

Par contre ce qu'on peut faire c'est d'automatiquement repérer les utilisateurs ayant commencé l'expé et ayant des champs vides pour les relancer. Par ex, un règle du style "Envoyer un email de relance à tous les utilisateurs ayant commencé une expé et ayant tel et tel champ vide après deux semaines du renseignement de l'XP", etc etc. Ça permet de savoir aussi sur quels champs les gens galèrent tout en fluidifiant leur processus d'entrée.

T'en penses quoi ?

**Reply by ErwanGv** [reply]
Merci de ton retour ! Effectivement je n'avais pas en tête le fait que si les champs sont obligatoires les utilisateurs ne peuvent pas sauvegarder.
Donc ta solution me va bien !

Erwan

**Comment by ErwanGv** [comment]
Bonjour Alejandro, 

J'ai reçu l'ensemble des retours utilisateurs, donc ci-dessous les dernières modifications et ça peut partir en prod !

Merci d'avance !

Erwan

Une remarque d'un professionnel sur l’accessibilité à l’auto-diagnostic/outil de déclaration à l’expérimentation : « Seules les cuisines satellites peuvent répondre à ce questionnaire puisqu'en me connectant sous le profil créé "Siresco", en tant que cuisine centrale livrant des repas à des cuisines satellites, je ne pouvais pas accéder à la rubrique. Est-ce normal ? »

Autodiagnostic
- [x] Autodiagnostic, 2e question : « Le menu végétarien proposé est : », reformuler les réponses en « un menu végétarien en plat unique, sans choix », « un menu végétarien composé de plusieurs choix de plats végétariens », « Un menu végétarien au choix, en plus d'autres plats non végétariens »

Questions avant les onglets « à date » et « en décembre 2022 »
- [x] Nouvelle question sous la date de lancement : « Avant la mise en place de l’expérimentation, mon établissement propose chaque jour : »,  réponses « Un menu unique pour tous les convives », « Une offre à choix multiples »
- [x] Nouvelle question sous la date de lancement : « L’option végétarienne quotidienne : », réponses « est proposée chaque jour aux convives librement »,  « fait l'objet d'une préinscription en amont ».

Questions dans les onglets « à date » et « en décembre 2022 »
- [x] Question "Pourcentage de menus végétariens servis par rapport aux autres menus", reformuler en "Quel est le taux de prise des menus végétariens par rapport aux menus non-végétariens en cas de choix multiple ?"

Question : "Dans votre plan alimentaire de 20 repas successifs, les options végétariennes servies dans votre établissement sont des plats à base de : 
- [x] Reformuler la question en "Les plans alimentaires sont établis sur 20 repas successifs, soit 4 semaines de 5 jours. Par exemple, une fréquence de 4/20 correspond à une fois par semaine, 8/20 à deux fois par semaine.... Dans votre plan alimentaire, les 20 options végétariennes servies sur 20 repas successifs sont à base de :"
- [x] modifier "Fromage (pané fromager, tartiflette sans lardons...) ..." en "Fromage (pané fromager, tartiflette sans lardons, ravioles au fromage...)" 
- [x] ajouter le choix "Féculents, légumes et sauces (pâtes sauce tomate, lasagnes végétariennes, riz ratatouille...)" (/20)
- [x] Ajouter une nouvelle question dessous : "Parmi les plats à base de céréales, quelle part représentent les céréales complètes et semi-complètes ?" (en %)

Gaspillage
- [x] Pour les cantines ayant déjà mis en place une option végétarienne quotidienne, et pour l'onglet "à date" uniquement, ajouter une question "Avez-vous constaté une évolution du gaspillage avec l'option végétarienne entre le moment de sa mise en place et aujourd'hui ?" avec comme réponse "Oui, le gaspillage des plats végétariens a diminué depuis la mise en place", "Oui, le gaspillage des plats végétariens a augmenté depuis la mise en place" "Non, le gaspillage des plats végétariens n'a pas évolué".
- [x] Question "Ecart du gaspillage en pourcentage", reformuler en "Ecart du gaspillage en pourcentage (ex : 30% de gaspillage en plus/en moins avec l'option végétarienne)".
- [x] Question « Résultats des pesées de gaspillage alimentaire sur au moins 3 jours consécutifs (en g/convive) », dans toutes les réponses, remplacer « Menus végétariens [non-végétarien] : » par « Plat principal végétarien [non-végétarien] », et mettre dans chaque réponse l’unité « g/convive » et non « g ».

Coût 
- [x] Introduire une question qualitative comme pour gaspillage et fréquentation, avant l'évaluation quanti : "En moyenne, le coût matière des plats végétariens est :" "Moins cher que les plats non-végétariens", "Plus que cher que les plats non-végétariens", "Equivalent aux plats non-végétariens"
- [x] Sous cette question, si la réponse est « Moins cher que les plats non végétariens », introduire une nouvelle question « Les économies réalisées ont été réinvesties pour augmenter la part de produits durables et de qualité (Bio, SIQO…) » (à cocher).
- [x] Pour les cantines ayant déjà mis en place une option végétarienne quotidienne, et pour l'onglet "à date" uniquement, ajouter une question "Avez-vous constaté une évolution du coût matière avec l'option végétarienne entre le moment de sa mise en place et aujourd'hui ?" avec comme réponse "Oui, le coût des plats végétariens a diminué depuis la mise en place", "Oui, le coût des plats végétariens a augmenté depuis la mise en place" "Non, le coût des plats végétariens n'a pas évolué".
- [x] Question "Comment le coût facturé aux familles a-t-il évolué depuis la mise en place du menu quotidien ? ", reformuler en « Y a-t-il eu une évolution du tarif par repas facturé aux familles depuis la mise en place de l’option végétarienne quotidienne ? » avec comme réponses possibles (plusieurs réponses possibles) : « Le tarif par repas n’a pas changé », « Le tarif par repas a augmenté », « Le tarif par repas a diminué », « Le tarif par repas est moins cher pour l’option végétarienne ».

Satisfaction
- [x] Question : "Quelles sont les principales raisons évoquées par les convives ?", ajouter le choix "Problèmes de digestion"
- [x] Question : « Satisfaction moyenne du personnel », reformuler en « Satisfaction moyenne du personnel (en cuisine et personnel encadrant) »
- [x] Question : "Quelles sont les principales raisons évoquées par le personnel ?", modifier "Méconnaissance" en "Méconnaissance (recettes, méthodes de préparation, utilisation du matériel)" et ajouter le choix "Manque de matériel adapté"

Autres questions
- [x] Pour les deux dernières questions, comme il s'agit d'une case à cocher il faudra reformuler avec une affirmation et pas une question : "Mon établissement a utilisé un livret de recettes végétariennes (CNRC ou autres organismes)" et "Mon établissement a mis en place une formation spécifique des cuisiniers ou gestionnaires sur les menus végétariens". De même, est-il possible d'ouvrir des champs libres lorsque que la case est cochée : "Précisez, quel livret de recette avez vous utilisé?" et "Précisez, quel type de formation ?"
- [x] Nouvelle question à la fin : « Quels sont les principaux freins rencontrés à la mise en place de l’option végétarienne quotidienne ? », réponses « Difficultés d'accès à la formation », « Difficultés d'approvisionnement », « Manque de recettes », « Surcoût des produits », « Réaction des convives », « Autre : (champ libre) ».
- [x] Nouvelle question à la fin : « La mise en place de l'option végétarienne a provoqué une surcharge de travail pour le personnel ? » (à cocher).

**Reply by alemangui** [reply]
Hello @ErwanGv, ça marche. On a mis cette story dans ce sprint. 

@JenniferStephan et @valeriemerle font une revue de ces changements, le dev commencera après cette revue. Si rien ne coince, semaine prochaine on pourrait les avoir en staging.

**Reply by alemangui** [reply]
Possible fusion entre : 

- Nouvelle question à la fin : « Quels sont les principaux freins rencontrés à la mise en place de l’option végétarienne quotidienne ? », réponses « Difficultés d'accès à la formation », « Difficultés d'approvisionnement », « Manque de recettes », « Surcoût des produits », « Réaction des convives », « Autre : (champ libre) ».
- Nouvelle question à la fin : « La mise en place de l'option végétarienne a provoqué une surcharge de travail pour le personnel ? » (à cocher).

**Reply by ErwanGv** [reply]
@alemangui , totalement d'accord avec toi, ça a plus de sens comme ça ! Merci de ta vigilance.

Erwan

**Comment by JenniferStephan** [comment]
Hello @ErwanGv,

On a balayé toutes les modifs; on aurait des questions à te poser sur celles que l'on a cochées. Serais-tu dispo lundi matin 9h30 pour en discuter tous les 4 ? Comme ça, cela part en dév pr la 2ème semaine de sprint.  

Concernant les cantines centrales, on est parti du principe qu'elles ne pouvaient pas faire d'auto-diag si elle n'avait pas de lieu de service. Le mieux donc, serait pour ton utilisateur de créer en masse les cantines satellites qu'il livre et on pourrait remplir de la même manière (en bloc) les résultats de l'expé. Est-ce que tu serais ok pour que l'on prévoit un créneau avec lui pour lui montrer la démarche ? 

merci bcp !!! :)

**Reply by ErwanGv** [reply]
Bonjour à tous,

Merci d'avoir pris le temps de regarder tout ça, ça sera compliqué à 9h30 lundi pour moi mais 10h c'est possible.
Concernant la question pour les cuisines centrales il s'agit de Magali Tempo, au SIRESCO (mtempo@siresco.fr).

Bonne journée,

Erwan

**Reply by JenniferStephan** [reply]
Merci de ta réponse @ErwanGv ! De mon côté, indispo à 10h mais @valeriemerle et @alemangui se connecteront : https://zoom.us/j/5516536304?pwd=bGRKK2haTlZhUk8xVStMRzd6Q0RIdz09 :)
L'idée seraient de prioriser les changements vus avec tes utilisateurs et éventuellement en supprimer quelques uns (nos propositions cochées à discuter avec toi) afin de ne pas allonger la liste de questions; déjà bien fournie :)
Plus si ok pour toi, se mettre en contact, par ton biais, je pense que ça serait mieux reçu, avec les membres de ton groupe qui auraient plusieurs cantines et pourrait accroître notre acquisition de nouveaux utilisateurs ! win-win :)

**Reply by alemangui** [reply]
Hello @ErwanGv  Concernant ce point : 

> Question "Comment le coût facturé aux familles a-t-il évolué depuis la mise en place du menu quotidien ? ", reformuler en « Y a-t-il eu une évolution du tarif par repas facturé aux familles depuis la mise en place de l’option végétarienne quotidienne ? » avec comme réponses possibles (plusieurs réponses possibles) : « Le tarif par repas n’a pas changé », « Le tarif par repas a augmenté », « Le tarif par repas a diminué », « Le tarif par repas est moins cher pour l’option végétarienne ».

Je vois deux soucis à mettre la quatrième option _« Le tarif par repas est moins cher pour l’option végétarienne »_ dans la même question : 
- En activant le choix multiple, on pourrait avoir des données contradictoires (par ex : _tarif augmenté_ et _tarif diminué_)
- Nous avons le champ conditionnel _Écart du coût facturé aux familles_ lorsque l'utilisateur indique un changement

Je te propose de garder la question avec les trois choix individuels (avec les nouveaux textes) et d'ajouter une case à cocher indépendante pour la quatrième :

![image](https://user-images.githubusercontent.com/1225929/167646944-61f29a71-fd9b-42ff-82a7-0b9de40d326a.png)

**Reply by ErwanGv** [reply]
Merci @alemangui , c'est effectivement la meilleure solution, ok pour moi !

**Comment by ErwanGv** [comment]
2 dernières micro-modifications avant de passer en prod (c.f. mail) :

- [x]  question "Quel est le taux de prise des menus végétariens par rapport aux menus non-végétariens en cas de choix multiple ? " -> modifier en "Quel est le taux de prise des menus végétariens par rapport aux menus non-végétariens en cas de choix multiple, **en moyenne sur 20 repas successifs** ?
- [x]  ajouter à la fin du questionnaire une case à cocher "Je souhaite être informé des conclusions de l'évaluation de l'expérimentation", pour envoyer aux participants le rapport d'évaluation quand il sera public.

**Reply by alemangui** [reply]
Hello @ErwanGv :wave:  Ces changements sont présents en staging. On passe en prod après ton :green_circle: _GO_

**Reply by ErwanGv** [reply]
Merci @alemangui  ! Je vous donne le go dès que c'est validé chez nous (a priori ça va remonter au cabinet, on vous tient au courant).
